README.md

```diff
@@ -0,0 +1,13 @@
+# General codecs
+
+This repository contains build scripts for general codecs:
++ [fix](codec-fix/README.md)
++ [itch](codec-itch/README.md)
++ [fast](codec-fast/README.md)
++ [ntg (native)](codec-ntg/README.md)
+
+The available parameters for each codec you can find in README files in codecs' folders.
+
+If you need to modify some parameters, please, change only parameters you need.
+
+_Do not specify default parameters if you don't want to override them_ (just because it doesn't make any seance to duplicate the configuration).
```

Comment:
_Do not specify default parameters if you don't want to override them_ (just because it doesn't make any **sense** to duplicate the configuration).